indico/modules/auth/auth_test.py

```diff
@@ -0,0 +1,47 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2023 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from datetime import datetime, timezone
+
+import pytest
+from flask import session
+from flask_multipass import IdentityInfo, Multipass
+from flask_multipass.providers.static import StaticIdentityProvider
+
+from indico.modules.auth import process_identity
+
+
+@pytest.mark.usefixtures('database')
```

Comment:
```suggestion
@pytest.mark.usefixtures('db')
```

database is session-scoped (creates the actual tables etc) and rather low-level while `db` does the proper setup of the sqlalchemy sessions, prevents data from being permanently committed to the test DB, etc.